multi-namespace-redb/README.md

```diff
@@ -58,10 +58,10 @@ roleRef:
 
 ```
 
-### 3. Updating the operator configmap
-
-The operator has to be deployed with a comma separated list of namespaces it will watch for REDB objects.
+### 3. Updating the managed namespaces
```

Comment:
Might be a good idea to mention that these two methods are mutually exclusive.
I'm not sure we would have a nice way to raise a visible error in this case (REC status?), but maybe we explicitly define precedence between the two methods if configured together.

Reply:
done